src/common/executor.ts

```diff
@@ -2,25 +2,27 @@
 // Licensed under the MIT license.
 
 "use strict";
-import { ChildProcess, exec, execSync, spawn, SpawnOptions } from "child_process";
+import { ChildProcess, exec, execSync, ExecSyncOptions, spawn, SpawnOptions } from "child_process";

```

Comment:
Minor: `exec` is imported but not used now.

Reply:
Thanks, will remove